src/main/java/seedu/duke/command/ListIngredientCommand.java

```diff
@@ -1,19 +1,30 @@
 package seedu.duke.command;
 
 import seedu.duke.data.Menu;
+import seedu.duke.data.dish.Dish;
 import seedu.duke.ui.Ui;
 
 /**
  * Lists all ingredients used in the selected dish to the user.
  */
-public class ListIngredientCommand extends Command{
+public class ListIngredientCommand extends Command {
     public static final String COMMAND_WORD = "list_ingredients";
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Lists out the ingredients needed along with the quantity for a specific dish.\n"
             + "Parameters: INDEX\n"
             + "Example: " + COMMAND_WORD + " 1";
+
+    public ListIngredientCommand(int listIndex){
```

Comment:
Perhaps add white space after bracket pleasee
```suggestion
    public ListIngredientCommand(int listIndex) {
```